src/components/ScholarshipList.test.jsx

```diff
@@ -44,8 +46,6 @@ beforeEach(() => {
 test('renders no results', async () => {
   renderWithProviders(<ScholarshipList />);
 
-  expect(screen.getByTestId('progress')).toBeInTheDocument();
```

Comment:
Oh, why was this removed?

Reply:
It was causing the test to fail. Im guessing it was because Suspense took over and rendered "loading"